### ISSUE TITLE
Doesn't load in Opera 11.11/Win7

### ISSUE THREAD

**Issue by teratorn** [fasterthanlime/jsmad#8]
Jsmad demo just displays "loading..."

**Comment by nddrylliog** [comment]
Yes, there's no Opera support at the moment. As far as I know, it doesn't
support the XMLHttpRequest features we're using, nor does it have any kind
of Audio API. When it does, ping us back so we can add support for it :)

Cheers,
Amos

On Sat, Jun 18, 2011 at 8:16 AM, teratorn <
reply@reply.github.com>wrote:

> Jsmad demo just displays "loading..."
> 
> ## 
> 
> Reply to this email directly or view it on GitHub:
> https://github.com/nddrylliog/jsmad/issues/8

## 

Amos Wenger
Community Development Engineer
official.fm - "The Do It Yourself Music Club"
I'm @nddrylliog https://github.com/nddrylliog on GitHub
